### PR TITLE
Add validations for all inputs

### PR DESCRIPTION
**Pull request by AbiudSM**
Ya no se puede romper el programa UwU

### CODE REVIEW

File: info_mutua_entropia.py [hunk 1]
```diff
@@ -1,36 +1,93 @@
...
 if(equiprobable):
-    prob = float(input(f'Ingrese la probabilidad de los eventos: '))
-    probabilidad_eventos.append(prob)
+	try:
+		prob = float(input(f'Ingrese la probabilidad de los eventos: '))
+		probabilidad_eventos.append(prob)
+
+		if (prob > 1) or (prob < 0):
+			print('Ingrese una probabilidad valida (entre 0 y 1)')
+			sys.exit()
```
Comment (Diegobr21) [inline info_mutua_entropia.py:71]: I think it would be better as: if(0<prob<1):

File: info_mutua_entropia.py [hunk 2]
```diff
@@ -1,36 +1,93 @@
...
 else:
-    for i in range(n):
-        prob = float(input(f'Probabilidad del evento {i}: '))
-        probabilidad_eventos.append(prob)
+	for i in range(n):
+		try:
+			prob = float(input(f'Probabilidad del evento {i}: '))
+			probabilidad_eventos.append(prob)
+
+			if (prob > 1) or (prob < 0):
```
Comment (Diegobr21) [inline info_mutua_entropia.py:83]: Same as above pls

**Review by Diegobr21 — DISMISSED** [review]
Nice job man! Thanks UwU

File: info_mutua_entropia.py (hunk 1 above)
Comment (AbiudSM) [reply info_mutua_entropia.py:71]: I couldnt do it u.u

**Comment by Robertudo** [comment]
Comment

**Review by Diegobr21 — APPROVED** [review]
Good to go

**Pull request merged**